planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -945,58 +833,42 @@ bool PullOverModule::hasFinishedPullOver()
 
 std::pair<HazardLightsCommand, double> PullOverModule::getHazardInfo() const
 {
-  HazardLightsCommand hazard_signal;
-  const double max_distance = std::numeric_limits<double>::max();
-
-  double distance_to_target_pose;   // distance from current pose to target pose on target lanes
-  double distance_to_target_point;  // distance from vehicle front to target point on target lanes.
-  {
-    const auto arc_position_target_pose =
-      lanelet::utils::getArcCoordinates(status_.pull_over_lanes, modified_goal_pose_);
-    const auto arc_position_current_pose =
-      lanelet::utils::getArcCoordinates(status_.pull_over_lanes, planner_data_->self_pose->pose);
-    distance_to_target_pose = arc_position_target_pose.length - arc_position_current_pose.length;
-    distance_to_target_point = distance_to_target_pose - planner_data_->parameters.base_link2front;
-  }
+  HazardLightsCommand hazard_signal{};
+
+  const auto arc_position_goal_pose =
+    lanelet::utils::getArcCoordinates(status_.pull_over_lanes, modified_goal_pose_);
+  const auto arc_position_current_pose =
+    lanelet::utils::getArcCoordinates(status_.pull_over_lanes, planner_data_->self_pose->pose);
+  const double distance_to_goal = arc_position_goal_pose.length - arc_position_current_pose.length;
 
   const auto velocity = planner_data_->self_odometry->twist.twist.linear.x;
   if (
-    (distance_to_target_pose < parameters_.hazard_on_threshold_dis &&
+    (distance_to_goal < parameters_.hazard_on_threshold_dis &&
      abs(velocity) < parameters_.hazard_on_threshold_vel) ||
```

Comment:
since `velocity` only used once here, it might be better to declare the `abs` in 
```
const auto velocity = fabs(planner_data_->self_odometry->twist.twist.linear.x);
```

Reply:
thaks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/1911/commits/c81e34e2d59b2aac75a65227bc70a68c4f45b809

Reply:
(Just comment. No need to fix in this PR)
When we use fabs or abs, we have to consider if the value is int or double. std::abs can deal with both int and double, so std::abs may be better.
http://blog.unnono.net/2014/02/abs.html

Reply:
wooow, this is horrible, my previous code had the mistake. than you for sharing.

Reply:
Owh, sorry @kosuke55-san, it is my bad. I just learned about this too. As @takayuki5168-san mentioned, since c++-14, `std::abs` have overloads that will able to handle both int and floating number.In addition, calling `abs` or `fabs` without `std` might invoke C-functions, therefore possibly affects the precision. However, if everyone agree it shouldn't make any differences, then there is no need to change them.

https://stackoverflow.com/a/61769735

Reply:
thaks! But I changed to used `std::abs`. https://github.com/autowarefoundation/autoware.universe/pull/1911/commits/b47d458c9870cfcb74d01d9866159bd273ee1972